src/components/Menu/MenuItem.tsx

```diff
@@ -78,16 +91,32 @@ export interface MenuItemProps
 
   /** Shorthand for the wrapper component. */
   wrapper?: ShorthandValue
+
+  /** Shorthand for the submenu. */
+  menu?: ShorthandValue
+
+  /** Indicates if the submenu is open */
+  submenuOpen?: boolean
```

Comment:
don't like this names duality - on the one side we have `menu`, on the other one `submenuOpen` - while, in fact, the same exact menu is implied by both. Suggest to rename to `menuOpen`